### PR TITLE
LPS-58391 - Convert frontend-themes-web respond-to mixins to media-qu…

### PR DESCRIPTION
**Pull request by pat270**
…ery mixin

https://issues.liferay.com/browse/LPS-58391

I made a cheat sheet for the media queries at https://gist.github.com/pat270/65b1adf31a356c282ef4

### CODE REVIEW

**Comment by natecavanaugh** [comment]
Heya Patrick,
Thanks for sending this over to me, and sorry for any confusion I might have caused.
I didn't want the code to necessarily output the same mediaqueries (unless we absolutely have to), but that I would prefer that we convert them in order to get the same effect.
For instance, on https://github.com/natecavanaugh/liferay-portal/pull/3462/files#diff-2036ed78dadc9c59911b451080e1bd76R21
couldn't we replace that with `@include md() {` instead of the `$breakpoint_tablet - 1` variable?

Basically, what I would like to do is convert to being able to use the `xs`, `sm`, `md`, and `lg` directly. But right now, that we have to have a conversion chart means it's ultimately not an improvement.

Does that make sense? Let me know if there's some aspect I'm not seeing though that might make it impossible.

Thanks Patrick,

**Comment by pat270** [comment]
Hey @natecavanaugh, I updated this pull with mobile first css. I had to add some things like https://github.com/pat270/liferay-portal/blob/433ef1b949a53f7b96f551535a8c9dbd126cb90d/modules/frontend/frontend-js-web/src/META-INF/resources/liferay/navigation.js#L19 and https://github.com/pat270/liferay-portal/blob/31df629ad708d8e16112a2379930d72eede2065b/modules/frontend/frontend-themes-web/src/META-INF/resources/classic/_diffs/css/_custom.scss#L22 to make things look better.

**Comment by natecavanaugh** [comment]
Merged and pushed to Brian. Thanks :)
Pull request submitted at: https://github.com/brianchandotcom/liferay-portal/pull/30276
View total diff: https://github.com/natecavanaugh/liferay-portal/compare/f201da38a6...d53f27bb82
